pkg/agent/cache/manager.go

```diff
@@ -363,6 +384,28 @@ func (m *manager) regEntriesHandler(wg *sync.WaitGroup) {
 
 }
 
+func (m *manager) rotateBaseSVIDHandler(frequency time.Duration, wg *sync.WaitGroup) {
+	defer wg.Done()
+	ticker := time.NewTicker(frequency)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			err := m.rotateBaseSVID()
+			if err != nil {
+				m.Shutdown(err)
+				return
+			}
+		case <-m.ctx.Done():
+			m.log.Debug("Done with rotateBaseSVIDHandler")
```

Comment:
This log message seems like it could be confusing. We could omit it altogether... or add a "starting" message or something

Reply:
:thumbsup: